src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActiveState.test.js

```diff
@@ -180,4 +189,104 @@ describe('elements/content-sidebar/ActiveState/activity-feed/ActiveState', () =>
             ).toBe(isCurrentVersion);
         },
     );
+
+    test("should have ActivityItem focused when it's activeFeedEntry", () => {
+        const wrapper = getShallowWrapper({
+            activeFeedItem: comment1,
+        }).dive();
+
+        expect(wrapper.find('[data-testid="comment"]').prop('isFocused')).toBe(true);
+    });
+
+    test('should have ActiveItem focused when AnnotationThead', () => {
+        const commentItemSelor = '[data-testid="comment"]';
+        const threadselector = '[data-testid="comment"] > [data-testid="activity-thread"]';
```

Comment:
Rather than repeating this in multiple tests, can we create a top-level helper function that takes the wrapper as an argument?

```
const getComment = (wrapper, index = 0) => wrapper.find('[data-testid="comment"]').at(index)